test/integration/scheduler/util.go

```diff
@@ -194,6 +167,10 @@ func createNodes(cs clientset.Interface, prefix string, res *v1.ResourceList, nu
 	return nodes[:], nil
 }
 
+func defaultNodeWrapper() *st.NodeWrapper {
```

Comment:
I prefer we don't hide the details in this function.
Each caller should define the resources they need in their tests.
I know this goes against having repeated code, but this is actually a good practice when writing tests.

Reply:
IMO `v1.ResourcePods` is a bit special, unlike zero cpu/mem, a zero Pods number would simply fail a NodeFit plugin in any case. Which is usually not what we want. So having a default `v1.ResourcePods` looks good to me. However, I'd prefer to set the default value on NodesWrapper side - which would also benefit UT (I did that way in another PR: https://github.com/kubernetes/kubernetes/pull/92571/files?file-filters%5B%5D=.go#diff-1cea28cd0be3cdbab57f5dc287dc98c0R367-R383)

If you do so, this function is not needed anymore - you can just replace callers of `defaultNodeWrapper()` directly to `st.MakeNode()`.

Reply:
Sg, we can just do `res[pods] = 32` at the beginning of the function, and the loop would override it.

Reply:
Yup, that would save one `if` block.

Reply:
Sounds good, I have made the change.